lib/ansible/plugins/lookup/credstash.py

```diff
@@ -65,10 +65,10 @@
   tasks:
 
   - name: "Test credstash lookup plugin -- get the password with a context passed as a variable"
-    debug: msg="{{ lookup('credstash', 'some-password', context=context) }}"
+    debug: msg="{{ lookup('credstash', 'some-password', context=**context) }}"
```

Comment:
this is invalid in a playbook

Reply:
How do you mean? I'm using this in a role